packages/eslint-config-hypothesis/package.json

```diff
@@ -9,10 +10,16 @@
   "scripts": {
     "test": "echo \"Warning: no test specified\" && exit 0"
   },
+  "devDependencies": {
+    "eslint": "^9.12.0",
+    "eslint-plugin-mocha": "^10.4.3",
+    "eslint-plugin-react": "^7.34.3",
+    "eslint-plugin-react-hooks": "^5.0.0"
+  },
```

Comment:
Yarn doesn't seem to install peer dependencies automatically, so I added them as dev dependencies as well.